src/assets/export_to_cloud.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.cls-1{fill:#fff;}</style></defs><title>export_to_cloud</title><path class="cls-1" d="M18.81,8.73A7.44,7.44,0,0,0,5.32,8.12,6,6,0,0,0,6,20H18.88a5,5,0,0,0,4.37-2.65A5.27,5.27,0,0,0,24,14.63,6,6,0,0,0,18.81,8.73Zm2.69,7.82A3.11,3.11,0,0,1,18.88,18H6a4,4,0,0,1-3.78-2.8h0a4,4,0,0,1,2.71-5A3.94,3.94,0,0,1,6,10.06h.71a5.46,5.46,0,0,1,10.62.47l0,.12H18a4,4,0,0,1,4,4A3.93,3.93,0,0,1,21.5,16.55Z"/><polygon class="cls-1" points="12.9 9.79 16 12.89 12.9 15.99 12.9 14.13 8 14.13 8 11.65 12.9 11.65 12.9 9.79"/></svg>
```

Comment:
Not sure why this is not wrapped but the `dark` one is. Either way is OK but it would be nice to be consistent.

Reply:
You're right, already fixed!